lib/iris/_lazy_data.py

```diff
@@ -232,17 +236,18 @@ def as_lazy_data(
         This will be converted to a :class:`dask.array.Array`.
     chunks : list of int, optional
         If present, a source chunk shape, e.g. for a chunked netcdf variable.
+        If set to "auto", Iris chunking optimisation will be bypassed, and dask's
+        default chunking will be used instead.
     asarray : bool, default=False
         If True, then chunks will be converted to instances of `ndarray`.
         Set to False (default) to pass passed chunks through unchanged.
+    meta : numpy.ndarray, optional
+        Empty ndarray created with same NumPy backend, ndim and dtype as the
+        Dask Array being created.
     dims_fixed : list of bool, optional
         If set, a list of values equal in length to 'chunks' or data.ndim.
         'True' values indicate a dimension which can not be changed, i.e. the
         result for that index must equal the value in 'chunks' or data.shape.
-    dask_chunking : bool, default=False
```

Comment:
I combined the functionality of the `dask_chunking` argument with the `chunks` argument (`dask_chunking = chunks == 'auto'`): https://github.com/SciTools/iris/blob/00af67eed9fce33a901d646d14034e7135d6bec9/lib/iris/_lazy_data.py#L237-L240
I did this because, with the addition of the `meta` argument, the function started to feel a bit bloated because of the many arguments. However, if you prefer the previous function signature, I would be happy to restore it.

Reply:
Ok, my apologies for being slow!
I can see that you have preserved the behaviour, and simplified the code too.
And I've now also realised that none of this is public API.